source/deployment/deployment.md

```diff
@@ -48,14 +47,37 @@ Typically a "Mattermost unreachable" error message will be displayed warning use
 
 WSS is a secure, encrypted connection and is highly recommended. An unencrypted WS connection may be used in initial testing and configuration but it is not recommended for production.
 
-## VPN Setup
+## Network Access and Multi-Factor Authentication 
 
-Mattermost offers "modern communication behind your firewall" and is designed deploy on your private network, with optional access from the outside over a Virtual Private Network (VPN) with multi-factor authentication enabled. This means running a VPN client on PC and mobile devices accessing Mattermost.
+While Mattermost is focused on private cloud deployment, there are a range of options for configuring network access for Team Edition (TE) and Enterprise Edition (EE). 
 
-To deploy a system outside your firewall, by opening standard ports such as 80 and 443, we recommend: 
+- In general, Mattermost recommends access via a Virtual Private Network client using at least a secondary authentication option, such as [OpenVPN](https://openvpn.net/), in addition to Mattermost authentication. 
+- For deployments accessible from the open internet, it is highly recommended to have dedicated staff managing system security and maintaining security updates in addition to enabling 2FA features in Mattermost Enterprise Edition. For non-enterprise deployments VPN is recommended. 
 
-1. Ensuring that an IT admin is subscribed to the [Mattermost security bulletin](https://about.mattermost.com/security-bulletin/) and able to apply any new security updates. 
-2. Upgrading to Mattermost Enterprise Edition for the ability to use multi-factor authentication without the MFA-enabled VPN client required for Team Edition deployments.
+While there are a multitude of different deployment configurations, the following outline popular configurations: 
+
+- **1. Private Network only with MFA (TE & EE)** - Access to Mattermost only available on private network. Multi-factor authentication includes securing access to computing devices along with physical security within buildings. Mobile web view used in place of Native Mobile Applications. Used in very high security, on-premises deployments. 
```

Comment:
Also, I don't know what the second sentence here is supposed to mean since it sounds like MFA has to do with physically securing the servers. Maybe "Includes physically securing access to computing devices within the building and using multi-factor authentication to verify identity."